guides/common/modules/proc_creating-a-webhook.adoc

```diff
@@ -0,0 +1,43 @@
+[id="creating-a-webhook_{context}"]
+= Creating a Webhook
+
+.Procedure
+
+. In the {Project} web UI, click *Administer* > *Webhooks*.
+. Click *Create Webhook*.
+. Click *Subscribe to field* to reveal a list of available events.
+. Select an event from the list.
+. Enter a name.
+. Enter a target URL. When using the shellhooks plugin, the URL should be in the form of `https://{smartproxy.example.com}:{smartproxy_port}/shellhooks/my_script`.
+. Click *Template* to see the list of available templates.
+. Select a template from the list.
+. Enter a HTTP method.
+. Ensure that the *Enabled* flag is checked if you want to create active webhook.
+
+In the Credentials tab:
+
+. Optional: Enter the username and password when HTTP authentication is required. Do not enter anything when using shellhooks.
+. Select *Verify SSL* if the server certificate should be verified against the system certificate store or Foreman CA. When using shellhooks, ensure this flag is selected.
+. Select *Proxy Authorization* when using shellhooks, otherwise deselect this box.
+. Optional: Provide one or more CA certificates in the X509 Certification Authorities. Keep this field blank when using shellhooks.
+
+In the Additional tab:
+
+. Enter the HTTP Content Type, typically application/json, application/xml or text/plain depending on the payload.
+. Optional: Provide HTTP headers as JSON. ERB is allowed and the same macros and variables are available as in webhook templates.
+
+When configuring webhooks with endpoints with non-standard HTTP(s) ports, SELinux port must be assigned, see  https://access.redhat.com/documentation/en-us/red_hat_satellite/6.9/html-single/installing_satellite_server_from_a_connected_network/index#configuring-selinux-to-ensure-access-on-custom-ports_satellite[Configuring SELinux to Ensure Access to Satellite on Custom Ports] in _Installing Satellite Server From A Connected Network_.
```

Comment:
We do not use (s) in IBM style: 

IBM style page 241:

Do not form a plural by adding (s). Try to rewrite the sentence to use either the plural
form or singular form, whichever is more appropriate. If you must indicate both forms,
repeat the noun, or use one or more. For example, instead of writing “enter the registra-
tion number(s),” write either “enter the registration number or numbers” or “enter one or
more registration numbers.”